### ISSUE TITLE
Genbank features are not parsed with attribtues.

### ISSUE THREAD

**Issue by Koeng101** [TimothyStiles/poly#152]
**Describe the bug**
```go
package main

import (
        "fmt"
        "github.com/koeng101/poly"
)

func main() {
        // Parse Genbank file
        sequence := poly.ReadGbk("data/Arthrospira_platensis.gb")
        for _, feature := range sequence.Features {
                if feature.Type == "CDS" {
                        fmt.Println(feature)
                }
        }
        // Find associated uniprot numbers

        // Find associated Rhea reaction numbers

        // Find associated Rhea
}
```
(koeng101/poly has up to date io.go file)

Output:
```
...
{  CDS    map[codon_start: inference: locus_tag: note: product: protein_id: transl_table: translation:] 6785219..6785560  {6785218 6785560 false false false false []}    0xc000014380}
{  CDS    map[codon_start: inference: locus_tag: note: old_locus_tag: product: pseudo: transl_table:] 6785764..>6785982  {6785763 6785982 false false false true []}    0xc000014380}
{  CDS    map[codon_start: inference: locus_tag: note: old_locus_tag: product: protein_id: transl_table: translation:] complement(6785992..6786264)  {6785991 6786264 true false false false []}    0xc000014380}
{  CDS    map[codon_start: inference: locus_tag: note: old_locus_tag: product: protein_id: transl_table: translation:] complement(6786272..6786772)  {6786271 6786772 true false false false []}    0xc000014380}
{  CDS    map[codon_start: inference: locus_tag: note: old_locus_tag: product: protein_id: transl_table: translation:] complement(6786769..6787014)  {6786768 6787014 true false false false []}    0xc000014380}
{  CDS    map[codon_start: inference: locus_tag: note: old_locus_tag: product: protein_id: transl_table: translation:] complement(6787063..6787290)  {6787062 6787290 true false false false []}    0xc000014380}
{  CDS    map[codon_start: inference: locus_tag: note: old_locus_tag: product: protein_id: transl_table: translation:] complement(6787259..6787876)  {6787258 6787876 true false false false []}    0xc000014380}
{  CDS    map[codon_start: inference: locus_tag: note: old_locus_tag: product: pseudo: transl_table:] complement(6788153..>6788335)  {6788152 6788335 true false false true []}    0xc000014380}
```
CDS features are not keeping their translations. I do not know why.

**Comment by Koeng101** [comment]
[Arthrospira_platensis.gb.gz](https://github.com/TimothyStiles/poly/files/6490330/Arthrospira_platensis.gb.gz)

**Comment by Koeng101** [comment]
I believe this is fixed in #125
